fedimintd/src/fedimintd.rs

```diff
@@ -328,11 +330,28 @@ impl Fedimintd {
                 .with_module_instance(MetaInit::kind(), MetaGenParams::default())
         };
 
-        if is_env_var_set(FM_USE_UNKNOWN_MODULE_ENV) {
+        let s = if is_env_var_set(FM_USE_UNKNOWN_MODULE_ENV) {
             s.with_module_kind(UnknownInit)
                 .with_module_instance(UnknownInit::kind(), UnknownGenParams::default())
         } else {
             s
+        };
+
+        if is_env_var_set(FM_ENABLE_MODULE_LNV2_ENV) {
```

Comment:
Do we even need an environment variable? Or is there a backwards compatibility concern?

Reply:
We do not want to ship the module in 0.4 yet.